lib/sourcebit.js

```diff
@@ -232,6 +245,10 @@ class Sourcebit {
     try {
       const data = await queue;
 
+      onTransformEndCallbacks.forEach(({ args, callback }) => {
+        callback({ ...args, data });
+      });
```

Comment:
@eduardoboucas didn't we want to call it after files have been written?

Recall, next.js target plugin needs it to notify the client only after files have been updated with new content